Scripts/BrightSign_Scripts/Download_Tracking.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import gitlab
+import glob
+import os
+import shutil
+import zipfile
+
+from subprocess import call
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-t", "--token",  required=True,  help="Provide a GitLab Authentication Token")
+parser.add_argument("-v", "--verbose", required=False, action='store_true', help="If used, this script will list the builds it's examining along with their status for users to check what was available")
+
+args = parser.parse_args()
+
+trackingZip         = "./Build_ARM64_Linux_release.zip"
+trackingUnzippedDir = "./Build_ARM64_Linux_release"
+
+searchCount = 500
+
+gitlabToken = args.token
+
+gl = gitlab.Gitlab('https://gitlab.ultrahaptics.com', private_token=gitlabToken)
+
+if os.path.isfile(trackingZip):
+	print("Removing previous pickler zip file")
+	os.remove(trackingZip)
+
+if os.path.isdir(trackingUnzippedDir):
+	print("Removing Unzipped Pickler Directory")
+	shutil.rmtree(trackingUnzippedDir)
+
+print("Downloading Artifacts from GitLab")
+
+# Tracking Project ID: 455
+# ARM Release Build Job ID: 485691
```

Comment:
do these artefacts expire?

Reply:
The version Alex had _was_ set not to expire but they have removed that now. Unfortunately this means that this build stage cannot work until Alex gets us the GitLab hosted copy of the release version, which I guess means holding off on this merge until then. 😞

Reply:
This has now been switched for an updated build which removes the concerning debug symbols but will persist on our GL. 👍